Makefile.PL

```diff
@@ -18,6 +18,7 @@ requires
   'IO::CaptureOutput'        => 0,
   'JSON::PP'                 => 0,
   'JSON::RPC'                => 0.96,
+  'JSON::RPC::Dispatch'      => 0,
```

Comment:
Does this line make any difference at all? JSON::RPC::Dispatch is installed by the same distribution as JSON::RPC which is already in the list. The JSON::RPC line declares a minimum version but the JSON::RPC::Dispatch line does not.

Reply:
It is required for Backend to run, but it was not installed when I had just replaced `libmodule-build-tiny-perl` with `Module::Build::Tiny`. I had to manually install it.

Reply:
Yeah, Backend uses JSON::RPC::Dispatch. JSON::RPC::Dispatch was introduced in JSON-RPC-1.00_02. Makefile.PL only requires JSON::RPC 0.96. The minimum version requirement for JSON::RPC should be bumped.